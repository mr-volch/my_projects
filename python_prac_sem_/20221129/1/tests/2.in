class C(metaclass=dump):
    def __init__(self, val, yt=3):
        self.val = val
        self.yt = yt
	

    def kiz(self, other, another, this=2, those=None):
        return self.val + other + (another or self.val) + self.yt + this + (those or this)

c = C(val = 10)
print(c.kiz(9,10,this=890))
print(c.kiz(9,another=10,those = 932087482))
