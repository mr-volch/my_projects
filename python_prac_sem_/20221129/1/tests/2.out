__init__: (), {'val': 10}
kiz: (9, 10), {'this': 890}
1812
kiz: (9,), {'another': 10, 'those': 932087482}
932087516
